docs/ch_blocks.adoc

```diff
@@ -20,22 +20,9 @@ message asking you to update the vectors.  Once these two vectors have
 been updated, you can use the block words.
 
 If you would like to play with some blocks, but don't have any
-hardware or are running Tali in a simulator, fear not!  Tali has a
-built-in RAM drive that can be accessed by running:
+hardware or are running Tali in a simulator, fear not! Tali has a
+built-in RAM drive that can be used on any hardware or simulator.  If using the
+c65 simulator that comes with Tali, persistant storage can be simulated using
+a file on the PC.
 
-----
-4 block-ramdrive-init
-----
-
-This reserves a chunk of ram with four blocks in it (numbered 0-3) which is
-enough to play around with. It also sets up the routines for reading and writing
-blocks in this ramdrive for you. If you want more blocks, you can change the
-number. Because they start at zero, the last valid block will always be one less
-than the number you provide.
-
-Be careful about creating too many blocks as they are 1K each.  It's also worth
-noting that running `block-ramdrive-init` again will create another ramdrive and
-the existing one will be inaccessible while still taking up space in
-RAM.
-
-See the tutorials on working with blocks for more information on how to use them.
\ No newline at end of file
+The following tutorial walks through some examples using 
```

Comment:
Is there something missing here?

Reply:
I rearranged that section a bunch and ended up removing a couple of the sections, but I haven't deleted the files yet because I still want their contents for shuffling things around.  ch_blocks.adoc and ch_editor_block.adoc will no longer be included as their content was moved into the tutorial itself (ch_tutorial_blocks.adoc).  I suspect this is leftovers of me trying to piece things together.  I'll be deleting those other files soon, as I think it's better with all of the material in the tutorial section.